packages/downloads/package.json

```diff
@@ -10,14 +10,19 @@
   "types": "dist/esm/index.d.ts",
   "license": "Apache-2.0",
   "dependencies": {
-    "@esri/arcgis-rest-feature-layer": "^2.14.1",
-    "@esri/arcgis-rest-portal": "^2.14.1",
-    "@esri/arcgis-rest-request": "^2.14.1",
```

Comment:
I'm glad to see these become `peerDepenencies`, but Unfortunately, this will manifest as breaking change b/c people will now need to install these.

Also, I'm considering consolidating this package w/ `hub-content` (#306), which would solve this issue.

I think we should update the versions, but leave them as `dependencies` for now.

Reply:
Done.